docs/ref/forms/renderers.txt

```diff
@@ -0,0 +1,161 @@
+======================
+The form rendering API
+======================
+
+.. versionadded:: 1.11
+
+Django form widgets are rendered using the Django template backend system.
+
+The form rendering process can be customized at several levels:
+
+* Widgets can specify custom template names.
+* Forms and widgets can specify custom renderer classes.
+* Built-in templates can be overridden within a project or application.
+
+.. _low-level-widget-render-api:
+
+The low-level render API
+========================
+
+Widget templates are stored in the ``django/forms/widgets`` path. A project
```

Comment:
The second sentence is slightly misleading. "A project" cannot by default override a widget template (unless the default form renderer is changed to `ProjectTemplateRenderer`), though an installed app can do so. I think just changing "A project" to "An installed app" would preserve the main point while being technically accurate.